### ISSUE TITLE
Migration rules for 1.3 release

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/graphing-quadratics#179]
#178 (add support for dynamic locale) will certainly result in many API changes. According to the PhET-iO spreadsheet, this sim needs to provide support migration from 1.2 to 1.3. So migration rules will be needed.

**Comment by samreid** [comment]
I'll reassign this when we get closer to that stage. Self-unassigning for now.

**Comment by zepumph** [comment]
Unassigning until we work on this sim.

**Comment by pixelzoom** [comment]
Migration rules for Molecule Polarity https://github.com/phetsims/molecule-polarity/issues/141 was quite the project. @samreid suggested that, while it's still fresh in my mind, I should do this sim.  @kathy-phet approved.  So I'll proceed.

**Comment by pixelzoom** [comment]
This sim went very quicky, ~1 hour. Almost everything could be reused from molecule-polarity-migration-rules.ts or natural-selection-migration-rules.ts.   Tested with `?strictMigrationRules`.

Completion is now blocked by:
* https://github.com/phetsims/phet-io-wrappers/issues/513.

**Comment by zepumph** [comment]
This is no longer blocked by https://github.com/phetsims/phet-io-wrappers/issues/513 (note it is still out for review though). You will likely need AboutDialog rules from molecule polarity.

**Comment by pixelzoom** [comment]
In the above commit, I added rule for element `aboutDialog` by copying them from molecule-polarity-migration-rules.ts.  This sim has no Options dialog, so I did not need to add rules for element `optionsDialog`.

**Comment by pixelzoom** [comment]
Getting this sim ready for dev testing is a task for June 9-29 iteration, so relabeling this issue accordingly.

**Comment by pixelzoom** [comment]
Since CT and git-hooks are not checking migration rules, they can be (and have been) broken.  So self-assigning to check migration rules just before dev testing, and just before creating the RC branch.

**Comment by pixelzoom** [comment]
Notes for QA:
* The previous version is https://phet-io.colorado.edu/sims/graphing-quadratics/1.2/
* If you encounter a migration problem, please create a new issue, linked to this issue.

**Comment by KatieWoe** [comment]
Things look ok from what we've seen.

**Comment by pixelzoom** [comment]
Leaving this assigned to me. It needs to be checked again just before creating the 1.3 release branch.

**Comment by pixelzoom** [comment]
I tested the Migration wrapper before creating the 1.3 release branch, and there were no problems.  Please test again for 1.3.0-rc.1, and close this issue if everything looks OK.

**Comment by KatieWoe** [comment]
Migration looked ok in rc.1

**Comment by pixelzoom** [comment]
Reopening. https://github.com/phetsims/graphing-quadratics/issues/202 will require API and migration rule changes, so this issue will need to be verified again for 1.3.0-rc.2.

**Comment by pixelzoom** [comment]
Ready for testing with the Migration wrapper in 1.3.0-rc.2.   Please assign back to me when testings is completed.

**Comment by KatieWoe** [comment]
Migration looked ok in 1.3.0-rc.2. Focused around point tool focus.

**Comment by pixelzoom** [comment]
This is not necessary to test in 1.3.0-rc.3, because the only issue to address is https://github.com/phetsims/graphing-quadratics/issues/203, which involved no changes to migration rules.

**Comment by pixelzoom** [comment]
Proceeding with publication of 1.3.0, so it's now safe to close this issue.
